plugins/labhub.py

```diff
@@ -213,6 +213,12 @@ def create_issue_cmd(self, msg, match):
 
         if repo_name in self.REPOS:
             repo = self.REPOS[repo_name]
+            issue_list = list(repo.search_issues())
```

Comment:
There will be thousands of issues in each repository, get the right ones and then check the body, title in those issues.

Reply:
@Vamshi99 Im not quite sure I understand what your saying, how do I get the "right ones", I would have to scan all the issues in order to compare the titles, else how would I determine the duplicate one .

Reply:
@sladyn98 You should have a look at the comments on the previous PR https://github.com/coala/corobo/pull/332 to get the correct approach of doing this.

Reply:
@abhishalya Yeah I did have a look at #332 . For every ```assign_cmd``` the corresponding repo would be selected and iterated through to determine a duplicate issue list.
Another approach I see would be to maintain a cache of issues and their titles, but IMO that would be a unnecessary overhead because of the constant updation.

Reply:
@sladyn98 Use a search query, it will give the issues which matches the text you gave, then search in those issues. See GitHub search and IGitt API if you don't know how.